Protocol.md

```diff
@@ -99,17 +70,73 @@ have no further frames sent and all frames will be ignored.
 
 ReactiveSocket as specified here only allows for TCP, WebSocket, and Aeron as transport protocols.
```

Comment:
I wouldn't bother with H2 streams in this document. It's going to be a deeper subject and best handled in its own doc.